spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -190,9 +222,24 @@ private Map setupPySparkEnv() throws IOException{
       }
     }
 
+    LOGGER.debug("PYTHONPATH: " + env.get("PYTHONPATH"));
     return env;
   }
 
+  // Run python shell
+  // Choose python in the order of
+  // PYSPARK_DRIVER_PYTHON > PYSPARK_PYTHON > zeppelin.pyspark.python
```

Comment:
hmm.. should the order be
`zeppelin.pyspark.python > PYSPARK_DRIVER_PYTHON > PYSPARK_PYTHON`

usually I think of zeppelin settings overriding what is set for spark